src/components/Button/readme.md

```diff
@@ -166,3 +152,17 @@ import { Button } from 'react-rainbow-components';
         <Button isLoading label="Button Brand" variant="brand" className="rainbow-m-around_medium" />
     </div>
 ```
+
+# Buttons of different sizes
+##### If you need to resize your button, you can do so using the `size` prop.
+
+```js
+import React from 'react';
+import { Button } from 'react-rainbow-components';
+
+    <div className="rainbow-p-vertical_large rainbow-align-content_center rainbow-flex_wrap">
+        <Button variant="brand" label="Button Small" className="rainbow-m-around_medium" size='small'/>
+        <Button label="Button Medium" variant="brand" className="rainbow-m-around_medium" />
```

Comment:
here I know the `medium` size is the default, but I think it is more clear to the user if we explicitly pass the ```size="medium"```